src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/medical/Medicine.java

```diff
@@ -21,6 +22,7 @@ public ItemConsumptionHandler getItemHandler() {
         return (e, p, item) -> {
             p.setFireTicks(0);
             clearNegativeEffects(p);
+            RadioactivityTask.radioactivityLevel.remove(e.getPlayer().getUniqueId());
```

Comment:
Do not ever access variables staticly unless they're constants

Reply:
Converting to functions